prov/usnic/src/usdf_msg.c

```diff
@@ -210,7 +210,50 @@ ssize_t
 usdf_msg_recvv(struct fid_ep *fep, const struct iovec *iov, void **desc,
                  size_t count, fi_addr_t src_addr, void *context)
 {
-	return -FI_ENOSYS;
+	struct usdf_ep *ep;
+        struct usdf_rx *rx;
```

Comment:
Watch the inconsistent indenting.  Please use actual tabs, per libfabric style (I think we have some older usnic/usnic_direct code that uses 4-space tabs, but we try to add all new code with libfabric-style actual tabs).